content/pages/changelog/index.md

```diff
@@ -13,6 +13,213 @@ For historical changelog entries, please see our
+
+#### Task update
+
+```js
+      {
+            "source": {
+                "item_type": "file",
+                "item_id": "123",
+                "item_name": "github_PNG20 (3) (2).png",
+                "parent": {
+                    "type": "folder",
+                    "name": "2020-02-27",
+                    "id": "1234"
+                },
+                "owned_by": {
+                    "type": "user",
+                    "id": "12345",
+                    "name": "Jane Doe",
+                    "login": "email@example.com"
+                }
+            },
+            "created_by": {
+                "type": "user",
+                "id": "12345",
+                "name": "Jane Doe",
+                "login": "email@example.com"
+            },
+            "action_by": null,
+            "created_at": "2020-04-02T22:26:55-07:00",
+            "event_id": "66585c35-97a8-4882-9fec-ce7e178b2e53",
+            "event_type": "TASK_UPDATE",
+            "type": "event",
+            "additional_details": {
+                "size": 67801,
+                "task": {
+                         "isCompleted": false,
+                         "due": "2020-07-06T10:49:44-07:00"
+"id": "1234567"
+"description": "task description"
+"creatorId": "567890"
+                },
+                "version_id": "26954851916",
+                "service_id": "12345",
+                "service_name": "Another App"
+            }
+        }
+```
+
+#### Task assignment delete
+
+```js
+        {
+            "source": {
+                "item_type": "file",
+                "item_id": "123",
+                "item_name": "football.png",
+                "parent": {
+                    "type": "folder",
+                    "name": "All Files",
+                    "id": "0"
+                },
+                "owned_by": {
+                    "type": "user",
+                    "id": "12345",
+                    "name": "Jane Doe",
+                    "login": "email@example.com"
+                }
+            },
+            "created_by": {
+                "type": "user",
+                "id": "12345",
+                "name": "Jane Doe",
+                "login": "email@example.com"
+            },
+            "action_by": null,
+            "created_at": "2020-05-28T15:53:28-07:00",
+            "event_id": "88c0f6fd-e644-468e-aac3-d8ad7fba4eff",
+            "event_type": "TASK_ASSIGNMENT_DELETE",
+            "ip_address": "Unknown IP",
+            "type": "event",
+            "session_id": null,
+            "additional_details": {
+                "size": 5875,
+                "version_id": "712094977889",
+                "service_id": "123",
+                "service_name": "Box Elements"
+            }
+        },
+```
+
+### Updated task and task assignment events with new fields
+
+#### Task assignment create
+
+The task assignment create event now includes task ID, assignee user ID and
```

Comment:
@cbetta I can revise this to be formatted more like `task.id`, `assigned_to.id`, etc if you would prefer that but I felt the human readable explanation of the updates with the example response was more consumable as a pair.

Reply:
Fine with me, as long as the actual value of the field keys is expressed in some other way (example, etc).